test/integration/__snapshots__/lerna-run.test.js.snap

```diff
@@ -41,13 +58,36 @@ lerna success - package-3
 lerna success - package-4"
 `;
 
+exports[`stderr: test --stream 2`] = `
+"lerna info version __TEST_VERSION__
+lerna success run Ran npm script 'test' in packages:
+lerna success - package-1
+lerna success - package-2
+lerna success - package-3
+lerna success - package-4"
+`;
+
 exports[`stdout: my-script --scope 1`] = `"package-1"`;
 
 exports[`stdout: test --ignore 1`] = `"package-4"`;
 
+exports[`stdout: test --stream --no-prefix 1`] = `
+"package-3
+package-4
+package-1
+package-2"
+`;
+
 exports[`stdout: test --stream 1`] = `
 "package-3: package-3
 package-4: package-4
 package-1: package-1
 package-2: package-2"
 `;
+
+exports[`stdout: test --stream 2`] = `
```

Comment:
I'm not sure where this snapshot is coming from? Or the `stderr: test --stream 2` addition.